static/panes/tree.ts

```diff
@@ -124,16 +124,19 @@ export class Tree {
         this.initCallbacks();
         this.onSettingsChange(this.settings);
 
-        this.selectize = new TomSelect(this.languageBtn, {
+        this.selectize = new TomSelect(this.languageBtn[0] as HTMLSelectElement, {
             sortField: 'name',
             valueField: 'id',
             labelField: 'name',
             searchField: ['name'],
-            options: _.map(usableLanguages, _.identity),
+            options: usableLanguages,
```

Comment:
This is the only change I'm not sure about. It seems like the map & identity were used at some point in any of the number of selectors we use, but it seems like its pourpose was to change from object-of-objects to array-of-objects.

In that case, this would not be needed here as usableLanguages is already an array